### PR TITLE
Gae Bulg translations

### PR DESCRIPTION
**Pull request by JayShee**

### CODE REVIEW

File: SKILL.tsv [hunk 1]
```diff
@@ -5614,8 +5614,8 @@ SKILL_20151102_005613	$Serpentine: Bleeding
 SKILL_20151102_005614	$Enemies hit by [Serpentine] have a 2% chance per attribute level to be afflicted with [Bleeding] for 8 seconds. Bleeding damage is proportional to the character's physical attack.
 SKILL_20151102_005615	$Gae Bulg: Enhance
 SKILL_20151102_005616	Increases the damage dealt on an enemy with [Gae Bulg] by 1% per attribute level.
-SKILL_20151102_005617	
-SKILL_20151102_005618
+SKILL_20151102_005617	$Create defense zone for 15 seconds where the spear is thrown. Increases physical defense and magical defense by 2% for party members within the zone.
```
Comment (Sunaries) [inline SKILL.tsv]: Creates a defense zone?

File: SKILL.tsv [hunk 2]
```diff
@@ -5614,8 +5614,8 @@ SKILL_20151102_005613	$Serpentine: Bleeding
...
+SKILL_20151102_005618	$Create attack zone for 15 seconds where the spear is thrown. Increases physical attack and magical attack by 2% for party members within the zone.
```
Comment (Sunaries) [inline SKILL.tsv]: Creates an attack zone?

**Comment by Sunaries** [comment]
:+1: Good to merge.

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**